system/ee/EllisLab/Addons/moblog/mod.moblog.php

```diff
@@ -932,7 +929,7 @@ function post_entry()
 
 		$channel_id = $this->moblog_array['moblog_channel_id'];
 
-		ee()->db->select('site_id, channel_title, channel_url, rss_url, comment_url, deft_comments, cat_group, field_group, channel_notify, channel_notify_emails');
+		ee()->db->select();
```

Comment:
Does an empty select have any effect here? The docs note to either omit it or use it as `ee()->db->select('*')` with the asterisk.

Reply:
`select()` works the same as `select('*')` - function is defined as `function select($select = '*', $escape = NULL)`
It could be omited indeed, but I've added it for better code readability